src/main/java/org/gridsuite/modification/server/dto/elasticsearch/EquipmentInfos.java

```diff
@@ -87,6 +87,23 @@ public static Set<VoltageLevel> getVoltageLevels(@NonNull Identifiable<?> identi
         throw NetworkModificationException.createEquipmentTypeUnknown(identifiable.getClass().getSimpleName());
     }
 
+    public static String getEquipmentTypeName(@NonNull Identifiable<?> identifiable) {
+        return identifiable.getType() == IdentifiableType.HVDC_LINE ? getHvdcTypeName((HvdcLine) identifiable) : identifiable.getType().name();
+    }
+
+    /**
+     * @param hvdcLine The hvdc line to get hvdc type name
+     * @return The hvdc type name string
+     * @throws NetworkModificationException if converter station types don't match
+     */
+    private static String getHvdcTypeName(HvdcLine hvdcLine) {
+        if (hvdcLine.getConverterStation1().getHvdcType() != hvdcLine.getConverterStation2().getHvdcType()) {
+            throw NetworkModificationException.createHybridHvdcUnsupported(hvdcLine.getId());
+        }
+
+        return String.format("%s_%s", hvdcLine.getType(), hvdcLine.getConverterStation1().getHvdcType());
```

Comment:
```suggestion
        return String.format("%s_%s", hvdcLine.getType().name(), hvdcLine.getConverterStation1().getHvdcType().name());
```
toString() should be used only for label/log

Reply:
done